lib/puppet/file_bucket/file.rb

```diff
@@ -79,6 +78,10 @@ def self.from_s(contents)
     self.new(contents)
   end
 
+  def self.from_binary(contents)
+    self.new(contents)
+  end
+
```

Comment:
yeah, sounds good. Maybe make a comment above the to_s methods that they are deprecated, `# @deprecated` so we can remove another day